docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -1,428 +1,86 @@
-
-The command has the following format:
-
-[source,shell]
-----
-docker run -p <ip_address_to_bind>:<host_port>:<container_port> -e HZ_CLUSTERNAME=source hazelcast/hazelcast:<source_version>
-----
-
-NOTE: The `-p` option in the above command maps the container's port to the host machine. This ensures that your Docker instance, which is running in a virtual network, is accessible to your local processes. The option is required because the migration and target clusters, CLC, and DMT run locally on your computer outside the Docker environment.
-
-For example, to start a version 4.2.7 source cluster on IP address 127.0.0.1 and port 5701, enter the following command in a terminal:
-
-[source,shell]
-----
-docker run -p 127.0.0.1:5701:5701 -e HZ_CLUSTERNAME=source hazelcast/hazelcast:4.2.7
-----
-
-==== Add Data to Cluster
-
-To access the cluster and populate it with data - for example, because you are using the DMT to test a migration of a Development cluster - you can do either of the following: 
- 
-* Use the _source.yaml_ configuration file, included in the _migration_config_ folder of the DMT download package
-
-* Write data to memory as described in the xref:getting-started:get-started-binary.adoc#step-3-write-data-to-memory[Step 3. Write Data to Memory] section of this documentation
-
-The _source.yaml_ file contains the following:
+* *General:*
 
-[source,yaml]
-----
-cluster:
-  name: "source"
-  address: "127.0.0.1:5701"
-----
-
-NOTE: If you have not installed the Hazelcast CLC, do this now. For further information on installing the CLC, refer to the https://docs.hazelcast.com/clc/latest/install-clc[Hazelcast Command-Line Client^] documentation.
-
-To make sure that you can add an entry to the source cluster, enter the following command in a terminal:
-
-[source,shell]
-----
-clc -c source.yaml map --name my-map set key-1 value-1
-----
-
-If an error relating to CLC being unable to connect to your source cluster is returned, confirm the following:
-
-* The port mapping is correct
-* The source cluster container is running
-* The configuration in your _source.yaml_ file is correct
-
-If no errors are returned, you can populate the source cluster with 1000 entries using the following script:
-
-[tabs] 
-==== 
-macOS and Linux:: 
-+ 
--- 
-[source,shell]
-----
-for i in {1..1000}; do clc -c source.yaml map --name my-map set key-$i value-$i --quiet; done && echo OK
-----
---
-
-Windows::
+** You can run only one migration at a time
+** Any empty data structures are not migrated
+** Indicative timings from our testing:
 +
-[source,shell]
-----
-for /l %x in (1, 1, 1000) do clc -c source.yaml map --name my-map set key-%x value-%x --quiet
-----
---
+[NOTE]
 ====
+These timings will be affected by entry and data structure counts as follows:
```

Comment:
Maybe an additional disclaimer around this part that mentions that these timings are just examples, and that individual migration times will depend on various factors including environmental constraints (i.e. network, hardware resources, etc.). Otherwise LGTM.